v-next/hardhat-zod-utils/src/index.ts

```diff
@@ -7,18 +7,50 @@ import { z } from "zod";
 /**
  * A Zod type to validate Hardhat's ConfigurationVariable objects.
  */
-export const configurationVariableType = z.object({
+export const configurationVariableType: z.ZodObject<
```

Comment:
hmm, having to define these types manually looks annoying. Maybe there's a better way, but I wouldn't worry until we work on the zod types.

Reply:
agreed